index.html

```diff
@@ -79,112 +79,9 @@ <h2 class="desktop-view-heading">My Recent Works</h2>
                     <img src="images/Indicator.png" alt="heading bar" height="4px" width="48px">
                 </div>
             </div>
-
-            <div class="work-item">
-                <div class="snapshot">
-                    <!--img tag should be here but figma template didn't have any image -->
-                </div>
-                <div class="details">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                    <ul class="tech-stack">
-                        <li>Ruby on Rails</li>
```

Comment:
- [ ] Please see the image below and **note** the portion _circled red_. After you dynamically render these project items, the section title `My recent works` and `See project` buttons _circled red_ on each card are behind the project item card. There should be proper spacing between cards and the section title. Kindly fix this

Your work
![image](https://github.com/batoolfatima2135/Portfolio/assets/84156714/3e4ff777-2d0a-417d-bf81-17e41ae4ba5c)

Figma design
![image](https://github.com/batoolfatima2135/Portfolio/assets/84156714/f44b9508-64b8-4bff-99c6-8f51ef5c243f)